### DISCUSSION TITLE
Scala Steward Github Action Not generating PRs

### DISCUSSION THREAD

**Discussion by yadavan88** [scala-steward-org/scala-steward#3184] [Q&A] [answered]
Hi,
I am here again with more question :) 

I have set up the Scala Steward Github action in one repository. However, it is not generating new PRs. 
In the logs, it shows the message for many libraries, for example:

```
2023-10-13 00:13:24,226 INFO  Found 36 updates:
    com.amazonaws:aws-java-sdk-s3 : 1.12.556 -> 1.12.566
    com.earldouglas:xsbt-web-plugin : 4.2.1 -> 4.2.4
    com.sksamuel.elastic4s:(elastic4s-client-esjava, elastic4s-client-esjava_2.13) : 8.9.3 -> 8.9.4
```

But again it shows the message that they are being ignored:
```
  2023-10-13 00:13:24,256 INFO  Ignoring outdated dependency com.amazonaws:aws-java-sdk-s3 for 1d 3m 36s
  2023-10-13 00:13:24,257 INFO  Ignoring outdated dependency com.earldouglas:xsbt-web-plugin for 1d 3m 36s
  2023-10-13 00:13:24,257 INFO  Ignoring outdated dependency com.sksamuel.elastic4s:elastic4s-client-esjava for 1d 3m 36s
  2023-10-13 00:13:24,257 INFO  Ignoring outdated dependency com.sksamuel.elastic4s:elastic4s-core for 1d 3m 36s
```
I am not sure what this means. I still have the Scala-Steward PRs generated from the scala-steward public job (where this repo is added to the markdown file).

Could someone help me to understand what is going wrong here? 

Here is the [repo](https://github.com/Baeldung/scala-tutorials/actions) if someone is interested to see the full logs. 

Thanks in advance

**Comment by mzuehlke** [comment, marked as answer]
In your `.scala-steward.conf` you have set the pull request frequency to **20 days**.
See: https://github.com/Baeldung/scala-tutorials/blob/master/.scala-steward.conf#L28

The logfile snippet you show above mentions that you have to wait 1 more day:
```
Ignoring outdated dependency XYZ for 1d 3m 36s
```
Or alter the configuration.

**Comment by yadavan88** [comment]
Thank you so much for the answer. I think last time it didn't create PRs due to the same PRs generated by public instance of scala-steward. 
I got confused and thought that the scala-steward conf file was used only by public instance.

I manually ran GHA and it created the PRs since 20 days period crossed. 

Thanks again!
